rules/windows/command_and_control_sunburst_c2_activity_detected.toml

```diff
@@ -16,7 +16,51 @@ license = "Elastic License v2"
 name = "SUNBURST Command and Control Activity"
 note = """## Triage and analysis
 
-The SUNBURST malware attempts to hide within the Orion Improvement Program (OIP) network traffic. As this rule detects post-exploitation network traffic, investigations into this should be prioritized."""
+### Investigating Remote File Download via Script Interpreter
+
+SUNBURST is a trojanized version of a digitally signed SolarWinds Orion plugin called
+SolarWinds.Orion.Core.BusinessLayer.dll. The plugin contains a backdoor that communicates via HTTP to third-party
+servers. After an initial dormant period of up to two weeks, SUNBURST may retrieve and execute commands that instruct
+the backdoor to transfer files, execute files, profile the system, reboot the system, and disable system services.
+The malware's network traffic attempts to blend in with legitimate SolarWinds activity by imitating the Orion
+Improvement Program (OIP) protocol and persistent state data is stored within legitimate plugin configuration files. The
+backdoor uses multiple obfuscated blocklists to identify processes, services, and drivers associated with forensic and
+anti-virus tools.
+
+More details on Sunburst can be found on the [Mandiant Report](https://www.mandiant.com/resources/sunburst-additional-technical-details).
```

Comment:
More of a question, thoughts on just adding this link in the references section?

Reply:
We can also add there, but the idea is to really point to the detailed research for analysts to get more context on